pkg/.bazelrc

```diff
@@ -0,0 +1,3 @@
+common --enable_platform_specific_config=true

```

Comment:
I'm worried about all of this.  Is this needed for rules_pkg or is it your own?

Reply:
Indeed after shell test is replaced with Python test .bazelrc is not needed anymore. Removed.